
public void run() throws IOException {
    
    int TEST_CASE = 1;
    while (TEST_CASE-- != 0) {
        
        
        
        
        
        
        
        
        
        long n = io.nextLong();
        long s = io.nextLong();
        io.out.println(func(n, s));
    
    }
}